index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="href='https://fonts.googleapis.com/css?family=Poppins" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+</head>
+
+<body>
+
+
+    <header>
+        <nav class="topnav">
+            <h2 class="logo">My Logo</h2>
+
+            <a class="icon">
+                <i class="fa fa-bars"></i>
+            </a>
+
+        </nav>
+    </header>
+    <section class="headline"></section>
```

Comment:
- [ ] You have closed the ```</section>``` tag immediately after opening 😨  I think this might be the cause of your background pic which is not showing on the page, it would be better to check well your codes 👍 
- [ ] Kindly try to add to your section a proper background image as given on the Figma design

   ### Your design
   ![image](https://user-images.githubusercontent.com/102757126/212126291-38a92dbb-78ee-407c-9bae-f612fe3d186e.png)
   
   ### Figma design
   ![image](https://user-images.githubusercontent.com/102757126/212126849-61a6dc4f-3dcb-422d-824e-9048f08fb294.png)